cluster-autoscaler/main.go

```diff
@@ -250,6 +251,7 @@ var (
 			"--max-graceful-termination-sec flag should not be set when this flag is set. Not setting this flag will use unordered evictor by default."+
 			"Priority evictor reuses the concepts of drain logic in kubelet(https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2712-pod-priority-based-graceful-node-shutdown#migration-from-the-node-graceful-shutdown-feature)."+
 			"Eg. flag usage:  '10000:20,1000:100,0:60'")
+	provisioningRequestsEnabled = flag.Bool("enable-provisioning-requests", false, "Whether the clusterautoscaler will be handling the ProvisioningRequest CRs.")
```

Comment:
nit: I'd avoid mentioning clusterautoscaler in the description. It is a flag to clusterautoscaler binary, so saying the flag will affect clusterautoscaler is a bit redundant.

Reply:
Yeah, the alternative is to put it in passive voice, but I don't think it's necessary. We already mention clusterautoscaler in many flags.